src/wobjectcpp.h

```diff
@@ -0,0 +1,124 @@
+#pragma once
+#include "wobjectdefs.h"
+
+namespace w_cpp {

```

Comment:
`cpp` does not mean much in this respect.
Maybe `w_builder` would be a better name?

Reply:
I'm not a fan of it either - just needed a name.
Maybe 
* `w_cpp` or `w_constexpr` to constrast it as C++ or Constexpr based API.
* `w_object` - to show the relation to W_OBJECT

Reply:
@ogoffart We still need to decide on the namespace name.
Do you have a preferred name?

Reply:
`w_object` is not bad.  since the macro is uppercase, it does not clash.
Since this allow to create properties, `w_builder` or `w_propertybuilder`
Or maybe `w_meta` ?